packages/graph/hash_graph/lib/graph/src/store/api/mod.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
I guess this intentionally was left blank and will replace the `Store` later on?

Reply:
I deleted this, even double checked 🙃. Weird rebase stuff I guess

Reply:
Should be removed